alma_api_cli/cli.py

```diff
@@ -0,0 +1,6 @@
+import click
+
+
```

Comment:
It just occurred to me that we might want to rename this directory since we don't know that all the cli commands are going to be related to the API. It's also nice to have a nice short, one-word command when possible. Feel free to use this opportunity to come up with a fun name like we did with hoard...can we backronym llama somehow?